stdlib/trains/trains.lua

```diff
@@ -105,7 +105,7 @@ function Trains._on_locomotive_changed()
         -- Rename it in the registry
         -- and dispatch a renamed event
         global._train_registry[renaming.new_id] = renaming.train
-        global._train_register[renaming.old_id] = nil
+        global._train_registry[renaming.old_id] = nil
```

Comment:
Could be worse :P If you compare the old defines.color to the newer defines.color you should be able to figure out my major goofup :1st_place_medal: